parsers/nmparser/parse_lst_file.go

```diff
@@ -5,28 +5,31 @@ import (
 	"strings"
 )
 
-// LstData is the output struct from a lst file
-type LstData struct {
-	RunDetails              RunDetails
-	FinalParameterEstimates FinalParameterEstimates
-	FinalParameterStdErr    FinalParameterEstimates
-	ParameterStructures     ParameterStructures
-	ParameterNames          ParameterNames
-	OFV                     float64
-}
-
-func parseOFV(line string) float64 {
-	// get rid of everything but stars and ofv
-	result := line[10:]
-	result = strings.Replace(result, "*", "", -1)
-	output, _ := strconv.ParseFloat(strings.Fields(result)[0], 64)
-	return output
+func parseOFV(line string, ofvDetails *OfvDetails) {
+	if strings.Contains(line, "#OBJV") {
+		// get rid of everything but stars and ofv
+		result := line[10:]
```

Comment:
what's on the first 10 characters that makes it worth trimming off?

Reply:
aka probably worth noting as a comment something like,

line should look like #OBJV: *** <value> ***, though with many more stars. So they can see that you're trimming the #OBJV

Reply:
This was existing code that I should have re-visited.  I've normalized this code so it follows the same pattern as the others.